### DISCUSSION TITLE
Unable to Zoom Out Beyond Current Limit in React Flow Application

### DISCUSSION THREAD

**Discussion by Cristhianzl** [xyflow/xyflow#3160] [Q&A] [answered]
I am currently working on a React Flow application where I need to implement the ability to zoom out beyond the current limit. However, I have encountered a limitation that prevents me from achieving the desired level of zoom out. I would appreciate any assistance or guidance on how to overcome this limitation and enable a greater zoom out functionality.

Description: In my React Flow application, I have successfully implemented zoom functionality, but I am unable to zoom out beyond a certain limit.

Current Limitation: The application restricts the zoom out capability to a specific threshold, which does not meet my requirements. I need to be able to zoom out further to have a broader view of the flow.
Expected Behavior: I would like to enhance the zoom out feature so that users can have the flexibility to zoom out to a level that suits their needs.

Attempts Made:
a. I have reviewed the React Flow documentation and explored the available zoom-related props and methods but haven't found a solution to increase the zoom out limit.
b. I have searched through the codebase and tried adjusting relevant properties or configuration options related to zoom, but without success.

Impact: The current limitation affects the usability and overall user experience of the application. Users are unable to zoom out to a level that allows them to see a wider perspective of the flow, potentially hindering their ability to work efficiently.

**Comment by chrtze** [comment, marked as answer]
Hey! You are probably looking for the `minZoom` prop: https://reactflow.dev/docs/api/react-flow-props/#minzoom

**Reply by Cristhianzl** [reply]
EDIT:

To get the maximum zoom out just minZoom={0.001} for example.
Thanks bro!

You saved the day :)

**Reply by mohitkumartoshniwal** [reply]
Thanks a lot for the value as well. Was struggling for an hour or so.
